src/Symfony/Component/Validator/Resources/translations/validators.lv.xlf

```diff
@@ -402,6 +402,30 @@
                 <source>The value of the netmask should be between {{ min }} and {{ max }}.</source>
                 <target>Tīkla maskas (netmask) vērtībai jābūt starp {{ min }} un {{ max }}.</target>
             </trans-unit>
+            <trans-unit id="104">
+                <source>The filename is too long. It should have {{ filename_max_length }} character or less.|The filename is too long. It should have {{ filename_max_length }} characters or less.</source>
+                <target>Faila nosaukums ir pārāk garš. Tas var būt { filename_max_length }} rakstzīmi vai īsāks.|Faila nosaukums ir pārāk garš. Tajā var būt {{ filename_max_length }} rakstzīmes vai īsāks.</target>
+            </trans-unit>
+            <trans-unit id="105">
+                <source>The password strength is too low. Please use a stronger password.</source>
+                <target>Paroles stiprums ir pārāk zems. Lūdzu, izmantojiet spēcīgāku paroli.</target>
+            </trans-unit>
+            <trans-unit id="106">
+                <source>This value contains characters that are not allowed by the current restriction-level.</source>
+                <target>Šajā vērtībā ir rakstzīmes, kuras nav atļautas pašreizējā ierobežojuma līmenī.</target>
```

Comment:
"Šajā vērtībā ir rakstzīmes" should be "Šī vērtība satur rakstzīmes" for closer translation of origina value